lib/server/web.js

```diff
@@ -39,8 +39,18 @@ function trimLocale(header) {
 // This is the webserver. It's what the outside always talks to. It
 // handles the whole Profile API.
 exports.create = function createServer() {
+  var useRedis = config.serverCache.useRedis;
+  var cache = {
+    engine: useRedis ? require('catbox-redis') : require('catbox-memory')
+  };
+  if (useRedis) {
+    cache.host = config.serverCache.redis.host;
+    cache.port = config.serverCache.redis.port;
+    cache.partition = config.serverCache.redis.partition;
+  }
   var isProd = config.env === 'production';
   var server = new Hapi.Server({
+    cache: cache,
```

Comment:
Since hapi uses memory cache by default anyway would it make sense to use the flag instead here to instruct the server method to not cache results if `useCach` is false: https://github.com/mozilla/fxa-profile-server/pull/253/files#diff-d63b253f4ac4a49ee08306581cd7b907R64 instead of overriding the default setting for the server's cache